website/templates/project/register_draft.mako

```diff
@@ -29,8 +29,9 @@
       <div id="register-show-submit">
 
         <hr />
-        
-        <p class="help-block">${language.BEFORE_REGISTRATION_INFO}</p>
+        <p class="help-block">
+          ${language.BEFORE_REGISTRATION_INFO | n}
```

Comment:
There is no HTML in this language string, so the `n` filter is not necessary here.